src/component/Legend.tsx

```diff
@@ -50,7 +50,7 @@ export type Props<TValue, TID> = DefaultProps<TValue, TID> & {
     right?: number;
   };
   payloadUniqBy?: UniqueOption<TValue, TID>;
-  onBBoxUpdate?: (box: ClientRect | DOMRect | null) => void;
+  onBBoxUpdate?: (box: DOMRect | null) => void;
```

Comment:
FYI: [`ClientRect` seems to be a pre-standard version of the canonical `DOMRect`](https://stackoverflow.com/a/65501184/270302).

Users who are referring to `ClientRect` probably need to change to `DOMRect` as well.